evalml/tests/pipeline_tests/test_time_series_baseline_pipeline.py

```diff
@@ -66,6 +66,12 @@ def test_time_series_baseline(
         time_index="date",
     )
     clf.fit(X_train, y_train)
+
+    # TODO: Replace this with a better test that reproduces the issue with bike_sharing
+    # causing the name of the internal X to shift.
+    X_train_t = clf._drop_time_index(X_train, y_train)
```

Comment:
Can someone file an issue to get some better testing around this?

Reply:
filed #3790 3790